tests/api/test_registry.py

```diff
@@ -101,6 +101,7 @@ def test_registry():
         "kafka-source-connector": KafkaSourceConnector,
         "kubernetes-app": KubernetesApp,
         "pipeline-component": PipelineComponent,
+        # TODO: change the old sterams bootstrap to -v2  and remove -v3
```

Comment:
Will you open a follow up PR?

Reply:
Yes I will :) I just don't want to make a branch from this PR and end up in rebase hell. I would rather do the reviews here. Merge it with v3 and then make a branch from v8 and continue with the TODOs.